lib/ui/view/threads/thread_detail/thread_detail_view.dart

```diff
@@ -22,218 +21,103 @@ class ThreadDetailView extends StatelessWidget {
     final _messageController = TextEditingController();
```

Comment:
This should not be here. Generate it that the stateless widget will go **with $ThreadDetailView**.
That file will give you the text controller that you need.

Achieving this is by making the view model of this page, **FormViewModel**.

Reply:
It doesn't require a form view model, I changed it to use hook widget's method